e2e/__tests__/empty-describe-with-hooks.test.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2018-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const path = require('path');
+const runJest = require('../runJest');
+const {extractSummary} = require('../Utils');
+const dir = path.resolve(__dirname, '../empty-describe-with-hooks');
+const ConditionalTest = require('../../scripts/ConditionalTest');
+
+ConditionalTest.skipSuiteOnJasmine();
+
+test('hook in empty describe', () => {
+  const result = runJest(dir, ['hook-in-empty-describe.test.js']);
+  expect(result.status).toBe(1);
+  expect(extractSummary(result.stderr)).toMatchSnapshot();
```

Comment:
missing the snapshots?

Reply:
Whoops. Got lost in the move to e2e. Added.